src/core/js/drawer.js

```diff
@@ -18,17 +18,13 @@ define([
         Adapt.trigger('drawer:triggerCustomView', view, hasBackButton);
     };
 
-    var init = function() {
-        var drawerView = new DrawerView({collection: DrawerCollection});
-
-        Adapt.on('app:languageChanged', function() {
-            drawerView.remove();
-            drawerView = new DrawerView({collection: DrawerCollection});
-        });
-    };
-
-    Adapt.once('adapt:start', function() {
-        init();
+    Adapt.on({
+        'adapt:start': function() {
+             new DrawerView({ collection: DrawerCollection });
```

Comment:
the extra space is still in

Reply:
I think it's my JS linter.  I actually think the spacing inside objects makes it more readable (see https://github.com/airbnb/javascript/#whitespace--in-braces).

Reply:
@oliverfoster's referring to the extra space in the indentation.